src/main/java/com/google/graphgeckos/dashboard/storage/DataRepository.java

```diff
@@ -32,7 +32,7 @@
    * @return true only if the operation completed successfully.
    * @throws IllegalArgumentException if entryData is null
    */
-  boolean createRevisionEntry(GitHubData entryData) throws IllegalArgumentException;
+  boolean createRevisionEntry(GitHubData entryData);
```

Comment:
it would be nice to add the `@NonNull` annotations to the interface too, for documentation (if that is allowed in Java)

Reply:
That's not allowed :(